app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1754,7 +1754,7 @@ table.orders{
   margin-top: 50px;
   font-family: 'SketchGothic', Arial, sans-serif;
   font-size: 58px;
-  color: white; 
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.